submissions/src/main/kotlin/gov/cdc/prime/reportstream/submissions/controllers/HealthController.kt

```diff
@@ -0,0 +1,13 @@
+package gov.cdc.prime.reportstream.submissions.controllers
+
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+class HealthController {
+
+    @GetMapping("/health")
+    fun health(): String {
```

Comment:
Can spring handle coroutines out of the box? Since we have a greenfield microservice we should try to use best practices (obviously not necessary for this current implementation but setting it as the standard will help us down the line).

Reply:
looks like they can... I'll update